src/TestSuite/DriverTestCase.php

```diff
@@ -140,13 +140,13 @@ public function testExportExecutableWithCompression()
 
         //No compression
         $result = $this->invokeMethod($this->Driver, '_exportExecutableWithCompression', ['backup.sql']);
-        $expected = $basicExecutable . ' > backup.sql' . REDIRECT_TO_DEV_NULL;
+        $expected = $basicExecutable . ' > \'backup.sql\'' . REDIRECT_TO_DEV_NULL;
```

Comment:
i think we need escapeshellarg() in the test suite as well to let it run on windows and linux successful (windows will escape using `"` while it's `'` on linux

Reply:
You're right.

Reply:
appveyor needs to be configured to not try to build something. a good example of proper configuration can be seen at https://github.com/sebastianbergmann/phpunit/blob/master/appveyor.yml

Reply:
if help is needed feel free to ask, or should i try to extend this pr to configure appveyor properbly?

Reply:
Yes, a PR is a good idea.